ledger/participant-integration-api/src/main/scala/platform/usermanagement/PersistentUserManagementStore.scala

```diff
@@ -48,6 +48,14 @@ final case class UserManagementConfig(
     maxUsersPageSize: Int,
 )
 
+final case class RateLimitingConfig(
```

Comment:
These classes would be  more at home somewhere nearer to the rate limiting mechanism itself. User management is rather unrelated.